content/en/resources/news.md

```diff
@@ -1,9 +1,14 @@
 ---
 title: News
-description:
-  The [press](resources/press) page contains a full listing of news coverage
+description: TUF news coverage
+# aliases: [/news/press]
```

Comment:
Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#`  comment it out?

Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

Reply:
> Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#` comment it out?
> 
> Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

Thanks for the review @lukpueh . I thought that was the Docsy syntax, but I'll wait for chalins input.

Reply:
> Can't remember the "front matter" syntax from the top of my head, but this does not seem to work. Does the `#` comment it out?
> 
> Also, IIUC, @chalin asked to make both "/news" and "/press" an alias for this page ( see #15)

so you mean have two alliases?